drivers/SmartThings/matter-sensor/fingerprints.yml

```
@@ -5,6 +5,11 @@ matterManufacturer:
     vendorId: 0x115F
     productId: 0x2002
     deviceProfileName: contact-battery
+  - id: 4447/8195
```

```suggestion
  - id: "4447/8195"
```
When the id is just a set of numbers, we usually add "quotes" around the numbers to ensure that they are interpreted as a string rather than as an equation.